PROTOCOL.md

```diff
@@ -153,18 +153,18 @@ represented by a space), and the rest of the string being their username.
 `|join|USER@STATUS` or `|j|USER@STATUS`
 
 > `USER` joined the room. Optionally, `USER` can end in `@` followed by a user status message.
-> A `STATUS` starting in `!` indicates the user is away.
+> `STATUS` will start in `!` if the user is away, or '` `' otherwise.
```

Comment:
I don't like this change.

I think there's a sense in which it's good that a user would be able to manually set an away status by adding `!` to the beginning of a status message.